src/table/TableHeadWrapper.jsx

```diff
@@ -46,12 +51,19 @@ function TableHeadWrapper({
       <TableRow className="sn-table-row">
         {tableData.columns.map((column, columnIndex) => {
           const tabIndex = columnIndex === 0 ? '0' : '-1';
+          const currentSortDir = SORT_NOTATIONS[column.sortDirection];
+          const isCurrentColumnActive = layout.qHyperCube.qEffectiveInterColumnSortOrder[0] === columnIndex;
+
           return (
             <TableCell
               key={column.id}
               align={column.align}
               className={`${classes.head} sn-table-head-cell sn-table-cell`}
               tabIndex={tabIndex}
+              role="button"
+              scope="col"
+              aria-sort={isCurrentColumnActive ? `${column.sortDirection}ending` : null}
```

Comment:
creative 👍

Reply:
🙏

Reply:
<img width="722" alt="Screenshot 2021-09-15 at 13 23 58" src="https://user-images.githubusercontent.com/4471489/133424919-29a70ff3-23ff-4e1a-ad92-196da753a08b.png">

got a warning message from eslint:
The attribute aria-sort is not supported by the role button.

Reply:
as far as i know, we should have put role as `columnheader` and that is the recommended way to handle it. But as I was talking with Max, the a360 side recommending us to have a role button for that since we have sorting going on there.